x-pack/legacy/plugins/security/server/routes/api/v1/authenticate.js

```diff
@@ -12,6 +12,27 @@ import { KibanaRequest } from '../../../../../../../../src/core/server';
 import { createCSPRuleString } from '../../../../../../../../src/legacy/server/csp';
 
 export function initAuthenticateApi({ authc: { login, logout }, config }, server) {
+  const xsrfWhitelist = server.config().get('server.xsrf.whitelist');
```

Comment:
__note:__ that the only BWC route deprecation approach I managed to find that wouldn't require NP plugin to expose SAML APIs directly (so that we can call them here). I don't like it, I'd rather be able to temporarily define routes with any URL for BWC reasons or have something like "URL alias" that we could define in NP for the new route. Will discuss with the Platform Team